src/applications/static-pages/widgetTypes.js

```diff
@@ -13,4 +13,5 @@ export default {
   POST_911_GI_BILL_STATUS_WIDGET: 'post-9-11-gi-bill-status',
   CORONAVIRUS_CHATBOT: 'va-coronavirus-chatbot',
   VIEW_DEPENDENTS_CTA: 'view-dependents-CTA',
+  FORM_686_CTA: '686UnauthContentBlock',
```

Comment:
It's kinda confusing having this `widgetType` named `FORM_686_CTA` and tied to the `vaViewDependentsAccess` feature, with there also being a dedicated `VIEW_DEPENDENTS_CTA` widget-type. Is there a relationship between the `VIEW_DEPENDENTS_CTA` and the `FORM_686_CTA`? Just wondering how we can help illustrate that at all.

Reply:
Yes, these are both tied to the same group in flipper. If the group sees one we want them to see both, make sense?